tests/foreman/api/test_rhci.py

```diff
@@ -0,0 +1,123 @@
+"""Unit tests for the ``activation_keys`` paths."""
+from ddt import data, ddt
+from fauxfactory import gen_string
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=R0904
+
+
+@ddt
+class RHCIDeploymentTestCase(APITestCase):
+    """Tests for the RHCI deployment API."""
+
+    @data(
+        gen_string(str_type='alpha'),
+        gen_string(str_type='alphanumeric'),
+        gen_string(str_type='cjk'),
+        gen_string(str_type='latin1'),
+    )
+    def test_positive_create_deployment(self, data):
+        """@Test: Create a simple deployment.
+
+        @Assert: An RHCI deployment is created with a random name.
+
+        @Feature: RHCI Deployment
+
+        """
+        try:
+            attrs = entities.RHCIDeployment(
+                name=data,
+                organization_id=1,
```

Comment:
Is it good to hard-code entity IDs like this? How about doing something like this:

```python
@classmethod
def setUpClass(cls):
    self.cls = entities.Organization().create()

@data(…)
def test_positive_create_deployment(self, data):
    attrs = entities.RHCIDeployment(
        name=data,
        organization=organization,
        …
    ).create_json()
```